test/built-ins/Array/prototype/concat/array-concat-arity.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2014 the V8 project authors. All rights reserved.
```

Comment:
Slightly different copyright than found in https://github.com/tc39/test262/blob/master/CONTRIBUTING.md#test-case-style